std/range/primitives.d

```diff
@@ -1670,6 +1670,8 @@ auto walkLength(Range)(Range range, const size_t upTo)
     that support slicing and have length.
     Completes in $(BIGOH n) time for all other ranges.
 
+    This is similar to `drop` (and `dropRight`) in other languages.
+
```

Comment:
Slightly misleading, since D also has a drop function. Better alternative would be to mention it's difference to drop(), and perhaps also add (if it has not already) a similar mention to description of drop().